config/logstash.yml

```diff
@@ -131,6 +131,15 @@
 #
 # queue.checkpoint.interval: 1000
 #
+# ------------ Dead-Letter Queue Settings --------------
+# Flag to turn on dead-letter queue.
+#
+# dead_letter_queue.enable: false
+#
+# If using dead_letter_queue.enable: true, the directory path where the data files will be stored.
```

Comment:
We should use the same strategy as PQ configuration. Mention in the comment here that DLQ default is `path.data/dead_letter_queue` and leave the setting value empty here.

See https://github.com/elastic/logstash/blob/master/config/logstash.yml#L96